docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
+
```

Comment:
Using NDK 13 (`ENV NDK_VERSION "r13b"`). Shouldn't this be upgraded? Current is at r15 and Q3 probably r17 is released.